### PR TITLE
Browser doesn't open for private distribution groups

### PR DESCRIPTION
**Pull request by AnastasiaKubova**
Please have a look at our [guidelines for contributions](https://github.com/microsoft/appcenter-sdk-android/blob/develop/CONTRIBUTING.md) and consider the following before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] ~Did you add unit tests?~
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?
* [x] Did you check UI tests on the sample app? They are not executed on CI.

## Description

Browser doesn't open for private distribution groups on Android 11.

## Related PRs or issues

[AB#83712](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/83712)

### CODE REVIEW

**Review by russelarms — APPROVED** [review]

File: sdk/appcenter-distribute/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -9,16 +9,12 @@
...
-        It should be added manually by developers to support old Android devices.
-
-        <uses-permission
-            android:name="android.permission.WRITE_EXTERNAL_STORAGE"
-            android:maxSdkVersion="18" />
-    -->
+    <queries>
+        <intent>
+            <action android:name="android.intent.action.VIEW" />
+            <data android:scheme="https"/>
```
Comment (MatkovIvan) [inline sdk/appcenter-distribute/src/main/AndroidManifest.xml:15]: Could you please add a comment with explanation why it's here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # App Center SDK for Android Change Log
 
+## Version 4.0.1 (Under development)
+
+### App Center Distribute
+
+* **[Fix]** Fix opening browser when use private distribute group on Android 11.
```
Comment (Jamminroot) [inline CHANGELOG.md:7]:
```suggestion
* **[Fix]** Fix browser opening when using private distribute group on Android 11.
```

File: sdk/appcenter-distribute/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -10,15 +10,16 @@
...
-        to avoid manifest merging issues. If an application requires this permission for all versions this
-        will lead to silently removing it for versions bigger than 18.
-        It should be added manually by developers to support old Android devices.
-
-        <uses-permission
-            android:name="android.permission.WRITE_EXTERNAL_STORAGE"
-            android:maxSdkVersion="18" />
-    -->
+     In Android 11 was added a lot of updates that improve privacy. Now a list of installed apps on user devices is hidden by default.
+     These changes need to make available browsers on user devices visible during checking updates for private distribution groups.
```
Comment (DmitriyKirakosyan) [inline sdk/appcenter-distribute/src/main/AndroidManifest.xml:14]:
```suggestion
     These changes make available browsers on user devices visible during checking updates for private distribution groups.
```

File: sdk/appcenter-distribute/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -10,15 +10,16 @@
...
-        Do not add the WRITE_EXTERNAL_STORAGE permission with custom attributes to library's manifest file
-        to avoid manifest merging issues. If an application requires this permission for all versions this
-        will lead to silently removing it for versions bigger than 18.
-        It should be added manually by developers to support old Android devices.
-
-        <uses-permission
-            android:name="android.permission.WRITE_EXTERNAL_STORAGE"
-            android:maxSdkVersion="18" />
-    -->
+     In Android 11 was added a lot of updates that improve privacy. Now a list of installed apps on user devices is hidden by default.
```
Comment (DmitriyKirakosyan) [inline sdk/appcenter-distribute/src/main/AndroidManifest.xml:13]:
```suggestion
     In Android 11 there were added a lot of updates that improve privacy. Now the list of installed apps on user devices is hidden by default.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,13 @@
 # App Center SDK for Android Change Log
 
+## Version 4.0.1 (Under development)
```
Comment (MatkovIvan) [inline CHANGELOG.md:3]: Please fix conflicts

File: sdk/appcenter-distribute/src/main/AndroidManifest.xml [hunk 6]
```diff
@@ -10,15 +10,16 @@
...
-        Do not add the WRITE_EXTERNAL_STORAGE permission with custom attributes to library's manifest file
-        to avoid manifest merging issues. If an application requires this permission for all versions this
-        will lead to silently removing it for versions bigger than 18.
-        It should be added manually by developers to support old Android devices.
-
-        <uses-permission
-            android:name="android.permission.WRITE_EXTERNAL_STORAGE"
-            android:maxSdkVersion="18" />
-    -->
+     In Android 11 there were added a lot of updates that improve privacy. Now the list of installed apps on user devices is hidden by default.
```
Comment (MatkovIvan) [inline sdk/appcenter-distribute/src/main/AndroidManifest.xml:13]:
```suggestion
     Android 11 added a lot of improving privacy updates. The list of installed apps on user devices is hidden by default now.
```

**Review by MatkovIvan — DISMISSED** [review]

File: CHANGELOG.md [hunk 7]
```diff
@@ -2,6 +2,10 @@
 
 ## Version 4.0.1 (Under development)
 
+### App Center Distribute
+
+* **[Fix]** Fix browser opening when using a private distribute group on Android 11.
```
Comment (russelarms) [inline CHANGELOG.md:7]:
```suggestion
* **[Fix]** Fix browser opening when using a private distribution group on Android 11.
```

File: sdk/appcenter-distribute/src/main/AndroidManifest.xml [hunk 8]
```diff
@@ -10,15 +10,16 @@
...
-        will lead to silently removing it for versions bigger than 18.
-        It should be added manually by developers to support old Android devices.
-
-        <uses-permission
-            android:name="android.permission.WRITE_EXTERNAL_STORAGE"
-            android:maxSdkVersion="18" />
-    -->
+     Android 11 added a lot of improving privacy updates. The list of installed apps on user devices is hidden by default now.
+     These changes make available browsers on user devices visible during checking updates for private distribution groups.
+     See more about managing package visibility: https://developer.android.com/training/basics/intents/package-visibility
```
Comment (Jamminroot) [inline sdk/appcenter-distribute/src/main/AndroidManifest.xml:15]: Not sure if I got this right, feel free to reject if so.
```suggestion
     Android 11 brings in a lot of changes regarding privacy. By default, list of installed apps is now hidden.
     Manifest block below make browsers on user devices acessible in order to let SDK check for updates in private update track.
     See more about managing package visibility: https://developer.android.com/training/basics/intents/package-visibility
```

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**